File: rotateLeft3/RotateLeft3.java
package rotateLeft3;

public class RotateLeft3 {
    //Dado um Array de tamanho 3, retorne esse mesmo Array com seus elemntos deslocados para a esquerda.
    //(Ex.: ([1,2,3]) -> [2,3,1]; ([5,11,4]) -> [11,4,5]).

    public int[] rotateLeft3(int[] nums) {
        int[] num1 = new int[nums.length];
        for (int i = 0; i < nums.length; i++) {
            int n = i+1;
            if (n >= nums.length) n = 0;
            num1[i] = nums[n];
        }
        return num1;
    }
}

File: Python/List-1/rotate_left3.py
"""
Dada uma lista numérica de tamanho 3 ("nums"), retorne-a com os seus elementos deslocados
para a esquerda. Ex.:(([1, 2, 3]) → [2, 3, 1]; ([5, 11, 9]) → [11, 9, 5]).
"""


def rotate_left3(nums):
    if len(nums) != 1:
        return nums[1:] + nums[:1]
    else:
        return nums


print(rotate_left3([1, 2, 3, 4, 5]))
